charts/timber-fluentd/templates/service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-fluentd
```

Comment:
Can we consistently use `{{ include "timber-fluentd.fullname" . }}` instead of `{{ .Release.Name }}` ?